rubin_sim/moving_objects/ooephemerides.py

```diff
@@ -502,34 +525,40 @@ def generate_ephemerides(
     ):
         """Calculate ephemerides for all orbits at times `times`.
 
-        This is a public method, wrapping self._convert_times, self._generateOorbEphs
-        and self._convertOorbEphs (which include dealing with oorb-formatting of arrays).
+        The returned ephemerides are a numpy array that can be grouped
+        by object or by time.
 
-        The return ephemerides are in a numpy recarray, with axes
-        - if by_object = True : [ephemeris values][object][@time]
-        (i.e. the 'ra' column = 2-d array, where the [0] axis (length) equals the number of eph_times)
-        - if by_object = False : [ephemeris values][time][@object]
-        (i.e. the 'ra' column = 2-d arrays, where the [0] axis (length) equals the number of objects)
+        If they are grouped by object (by_object = True), the array
+        is organized as `ephemeris_values[object][time]`.
+        Here the "ra" column is a 2-d array where the [0] axis
+        length equals the number of ephemeris times.
 
-        The ephemeris values returned to the user (== columns of the recarray) are:
-        ['delta', 'ra', 'dec', 'magV', 'time', 'dradt', 'ddecdt', 'phase', 'solarelon', 'velocity']
-        where positions/angles are all in degrees, velocities are deg/day, and delta is the
-        distance between the Earth and the object in AU.
+        If they are grouped by time (by_object=False), the array
+        is organized as `ephemeris_values[time][object]`.
+        Here the "ra" column is a 2-d array where the [0] axis length
+        equals the number of objects.
+
+        All returned positions and angles are in degrees, velocities
+        are degrees/day and distances are in AU.
 
         Parameters
         ----------
-        ephtimes : `np.ndarray`
-            Ephemeris times in oorb format (see self.convertTimes)
+        ephtimes : `np.ndarray`, (N,)
```

Comment:
There is a mismatch here between the name in the docstring and in the call signature, which could be a problem if a user passes it in, say, a **kwargs dictionary.

Reply:
Fixed.